generated/spryk_argument_list.yml

```diff
@@ -17,6 +17,10 @@
   name: moduleSrcDirectory
   mode: 2
   description: 'moduleSrcDirectory argument'
+-
+  name: paymentMethod
+  mode: 2
+  description: 'paymentMethod argument'
```

Comment:
is this argument visible for any spryk command? if yes, maybe it should be something with more generic name?

Reply:
each argument visible only if it was called in spryks, paymentMethod argument called only in AddModulePayment, so no reason here to have more detailed desc